pay-api/src/pay_api/models/eft_credit_invoice_link.py

```diff
@@ -37,13 +37,17 @@ class EFTCreditInvoiceLink(BaseModel):  # pylint: disable=too-few-public-methods
     __mapper_args__ = {
         'include_properties': [
             'id',
+            'amount',
             'created_on',
             'eft_credit_id',
-            'invoice_id'
+            'invoice_id',
+            'status_code'
         ]
     }
 
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
     invoice_id = db.Column(db.Integer, ForeignKey('invoices.id'), nullable=False, index=True)
     eft_credit_id = db.Column(db.Integer, ForeignKey('eft_credits.id'), nullable=False, index=True)
+    amount = db.Column(db.Numeric(19, 2), nullable=True)
+    status_code = db.Column('status_code', db.String(25), nullable=False, index=True)
```

Comment:
The link status and amount will allow for pending payments. This will also help with any reversals / refunds when we know the specific amounts to undo and restore the EFT Credit.